extensions/adobe/experience/content-intelligence/webasset.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/content-intelligence/webasset",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Content Intelligence Web Asset",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Asset details associated to a web element",
+  "definitions": {
+    "webasset": {
+      "properties": {
+        "xdm:id": {
+          "title": "Asset Id",
+          "meta:immutable": true,
+          "meta:usereditable": false,
+          "type": "string",
+          "description": "A unique identifier given to every addressable asset in a given repository."
+        },
+        "xdm:name": {
+          "meta:usereditable": false,
+          "title": "Asset name",
+          "type": "string"
+        },
+        "xdm:path": {
+          "meta:usereditable": false,
+          "title": "Asset path",
+          "type": "string"
+        },
+        "xdm:format": {
+          "meta:usereditable": false,
+          "type": "string",
+          "title": "Asset format",
+          "pattern": "\\w+\\/[-.\\w]+(?:\\+[-.\\w]+)?",
+          "examples": [
+            "application/vnd.adobe.photoshop"
+          ],
+          "description": "The physical or digital manifestation of the resource. Typically, Format should include the media-type of the resource. Format may be used to determine the software, hardware or other equipment needed to display or operate the resource. Recommended best practice is to select a value from a controlled vocabulary (for example, the list of [Internet Media Types](http://www.iana.org/ assignments/media-types/) defining computer media formats)."
+        },
+        "xdm:modifyDate": {
+          "type": "string",
+          "format": "date-time",
+          "meta:usereditable": false,
+          "examples": [
+            "2004-10-23T12:00:00-06:00"
+          ],
+          "description": "The server date and time when the resource was last modified in the repository, such as when a new version of an asset is uploaded or a directory's child resource is added or removed. The date time property should conform to ISO 8601 standard. An example form is \"2004-10-23T12:00:00-06:00\"."
+        },
+        "xdm:url": {
+          "title": "Url",
+          "type": "string",
+          "format": "uri",
+          "description": "The absolute URL of the asset."
+        },
+        "xdm:tags": {
```

Comment:
should this be `xdm:keywords`?

Reply:
not really. we are using AI generated tags with a confidence score

Reply:
Ah, then in that case it's `xmp:machineKeywords`